css/components.css

```
@@ -20,50 +20,267 @@
+  border-color: var(--primary-color);
+}
+
+.stat-box--primary .stat-box__icon {
+  font-size: var(--icon-size-primary);
+  opacity: 1;
+}
+
+.stat-box--primary .stat-box__number {
+  font-size: var(--number-size-primary);
+  font-weight: 800;
+  font-size: clamp(1.8rem, 4vw, var(--number-size-primary));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)